plugins/paymethod/paypal/PaypalPaymentPlugin.php

```diff
@@ -176,6 +178,12 @@ public function isConfigured($context)
      */
     public function handle($args, $request)
```

Comment:
This will prevent responses from PayPal from being handled by OJS, but doesn't prevent OJS from creating transactions with PayPal. See also `PaypalPaymentForm.php` in the `display` function.